### DISCUSSION TITLE
[Improvement] Default reviewers for a Pull Request

### DISCUSSION THREAD

**Discussion by alexeisolodovnicov-tomtom** [community/community#16902] [Pull Requests]
**Background:**
Working in an organization, each developer is part of a team. Usually for each pull request there is a need to add some reviewers, and those reviewers are in 99% the same people (from developer's team). Currently for each Pull Request developer needs to search the reviewers by name and add them one by one.

**Improvement:**
Would be nice to have a possibility to set up default reviewers, so that for each new PR developer could already have the default reviewer list.

**Comment by mlandisbqs** [comment]
The time it takes to add reviewers to every PR adds up. I too would really like the ability to default reviewers... the simplest solution would be to default reviewers to the same folks that were reviewers on **the author's most recent pr**.  If a CODEOWNERS file is present, that list could be used **without the current requirement to enforce approvals before merging**.  I'm not sure why these options are coupled.
